### PR TITLE
Feature/edit profile

### PR DESCRIPTION
**Pull request by wilcoxva**

### CODE REVIEW

**Review by tommycoleman87 — APPROVED** [review]
This looks great!

File: src/api/index.js [hunk 1]
```diff
@@ -84,6 +103,7 @@ export {
   getExampleData,
   getProfileData,
   getDSData,
+  editProfileData,
   getUserProfileData,
   getAuthHeader,
 };
```
Comment (TylerAlsop) [inline src/api/index.js:109]: Nice changes

File: src/components/pages/EditProfile/EditProfile.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState, useMemo } from 'react';
```
Comment (TylerAlsop) [inline src/components/pages/EditProfile/EditProfile.js:1]: Very clean and readable. I like it.

File: src/components/pages/EditProfile/RenderEditCustomerProfile.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      <p>
+        <Link to="/">Home</Link>
+      </p>
+      <div className="form-container">
+        <Form onFinish={submitHandler}>
+          <Form.Item
+            rules={[
+              {
+                required: true,
+                message: 'Please input your name!',
```
Comment (TylerAlsop) [inline src/components/pages/EditProfile/RenderEditCustomerProfile.js:41]: I like this message. :)

**Review by TylerAlsop — APPROVED** [review]
Everything looked very clean and straight forward. I did not find any issues that I know of.
Thank you for cleaning up all of the filler code from the original code base.
There are a bunch of check warnings about items that "are defined but not used" that we may want to fix in the future just for cleanliness. Other than that everything looked good to me.

**Pull request merged**